docs/legal_hold.md

```diff
@@ -0,0 +1,139 @@
+Legal Holds Policy
+==================
+
+Legal Hold Policy information describes the basic characteristics of the Policy,
+such as name, description, and filter dates. It is important to note that the legal hold object contains no information about what this policy applies to.
+
+If an order of discovery is received or the customer is part of an ongoing litigation, a legal hold policy can be created to keep track of everything that needs to be held. The actual holding is done via the Legal Hold Assignments endpoint. When the holds are no longer needed, the policy can be released by calling DELETE.
```

Comment:
The "calling DELETE" at the end of this line doesn't make much sense in this context; consider just removing it.